src/neuroconv/tools/nwb_helpers/_configuration_models/_zarr_dataset_io.py

```diff
@@ -26,17 +26,14 @@
         "fixedscaleoffset",  # enforced indirectly by HDMF/PyNWB data types
         "base64",  # unsure what this would ever be used for
         "n5_wrapper",  # different data format
+        "pcodec",  # is erroneously imported before numcodecs 0.15, see https://numcodecs.readthedocs.io/en/stable/release.html?utm_source=chatgpt.com#id9
     )
 )
 
 # Forbidding lossy codecs for now, but they could be allowed in the future with warnings?
 # (Users can always initialize and pass explicitly via code)
 _available_zarr_codecs = set(_base_zarr_codecs - _lossy_zarr_codecs - _excluded_zarr_codecs)
 
-# TODO: would like to eventually (as separate feature) add an 'auto' method to Zarr
```

Comment:
Removed it because it belongs to an issue.

Reply:
which issue is it?

Reply:
There is no issue at the moment.

Reply:
Could you open an issue for this and link it here? I just want to make sure that it doesn't get lost.

Reply:
I think I expressed myself badly. I am removing a comment that is a feature request because it does not belong there.

I honestly believe we are too far from being able to do something like in a reasonable way but if you are enthusiastic about it feel free to open an issue request.

Reply:
See #1384

Reply:
https://github.com/catalystneuro/neuroconv/issues/1384.

Marking this as resolved.

Reply:
Sorry for double posting, refresh on open tab.situation.